### PR TITLE
feat(kitchen): run it via travis maybe

### PR DESCRIPTION
**Pull request by jguice**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
+
+sudo: true
+
+before_script:
+- source <(curl -sL https://raw.githubusercontent.com/zuazo/kitchen-in-travis/0.3.0/scripts/start_docker.sh)
+
+script:
+- bundle exec rake travis:ci
+# Run test-kitchen with docker driver, for example:
+- KITCHEN_LOCAL_YAML=.kitchen.docker.yml bundle exec kitchen test
```
Comment (mburns) [inline .travis.yml]: Where does `.kitchen.docker.yml` come from? Or is it just for optional overrides?

Comment (jguice) [inline .travis.yml]: Yeah, not sure...need to RTFM a bit more to get this working. ;)

**Comment by jguice** [comment]
https://github.com/zuazo/kitchen-in-travis is what I was shooting for FWIW

**Comment by mburns** [comment]
I'm now :-1: specifically on the Docker-in-Travis setup. We can instead wire up CI (CIrcle, at the moment) to make a Rackspace VM using `kitchen-rackspace` plugin.

This avoids needing any docker inception hacks and we have a free budget to spin up internal VMs. :smile:

**Comment by jguice** [comment]
yay!

**Pull request closed without merging**